src/main/java/stsjorbsmod/powers/FragilePower.java

```diff
@@ -16,23 +19,41 @@ public FragilePower(final AbstractCreature owner, final int turnsUntilSnap) {
 
         this.owner = owner;
         this.amount = turnsUntilSnap;
+        this.amount2 = MemoryManager.forPlayer(owner).countCurrentClarities();
+        this.type = SPECIAL;
```

Comment:
I feel uneasy about using the power bar to represent this; in the past when we used a relic the feedback we got regularly was "it's confusing for quicksilver to not be affecting the 7-turn count on the 'relic'", and I would expect to get the same feedback about time eddy not interacting with Fragile.

Reply:
Maybe a compromise here (especially now that there's a tutorial tip pointing to the snap counter UI) could be to still use the snap counter (instead of a power icon) as the UI primary, but use it consistently for Determination as well? ie, make Determination cause the snap counter to go rainbow immediately and special case the snap counter to say "at end of your turn" in the Determination case?

I think the health bar update and the dynamic tooltip are both good regardless of the power icon, so maybe keep those somehow (either with patches against the SnapCounter or with an InvisiblePower)? It's just the power icon in the buff/debuff bar that concerns me; I expect putting an icon there to cause players to get confused about Time Eddy not applying to it.